<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>Ggplot on</title><link>/tags/ggplot/</link><description>Recent content in Ggplot on</description><generator>Hugo</generator><language>en</language><lastBuildDate>Wed, 29 Jun 2016 14:20:55 +0000</lastBuildDate><atom:link href="/tags/ggplot/index.xml" rel="self" type="application/rss+xml"/><item><title>Fixing the problem of matplotlib and ggplot not showing graphs in python</title><link>/posts/fixing-the-problem-of-matplotlib-and-ggplot-not-showing-graphs-in-python/</link><pubDate>Wed, 29 Jun 2016 14:20:55 +0000</pubDate><guid>/posts/fixing-the-problem-of-matplotlib-and-ggplot-not-showing-graphs-in-python/</guid><description>While trying to make some graphs for one of my side projects recently, I came across an issue that both matplotlib and ggplot did not show up the graphs. I was able to save the graphs to an image file just fine, but if I tried to show them directly on screen while running the script, it just printed a number and exited. After tearing apart my hair for some time, I checked the backend being used by matplot lib:</description></item></channel></rss>